R/system.R

```diff
@@ -254,6 +249,19 @@ queryMongoDB <- function(host, port, database, collection, username, password, q
       url = stringr::str_c(url, "?authSource=", authSource)
     }
   }
+  return (url)
+}
+
+
+#' @export
+queryMongoDB <- function(host, port, database, collection, username, password, query = "{}", isFlatten, limit=100000, isSSL=FALSE, authSource=NULL, fields="{}", sort="{}", skip=0){
```

Comment:
could you set `limit=0` as a default ? LIn2 wants to use (all i.e. 0) as a default value.